talpid-dbus/src/network_manager.rs

```diff
@@ -574,8 +574,11 @@ impl NetworkManager {
         settings: Settings,
         version_id: u64,
     ) -> Result<()> {
-        self.as_path(device)
-            .method_call(NM_DEVICE, "Reapply", (settings, version_id, 0u32))?;
+        self.as_path(device).method_call::<(), _, _, _>(
+            NM_DEVICE,
+            "Reapply",
+            (settings, version_id, 0u32),
+        )?;
```

Comment:
This could also work, and is slightly shorter
```suggestion
        let _: () =
            self.as_path(device)
                .method_call(NM_DEVICE, "Reapply", (settings, version_id, 0u32))?;
```